atomic_reactor/plugins/pre_resolve_composes.py

```diff
@@ -110,8 +110,9 @@ def run(self):
             self.wait_for_composes()
         except WaitComposeToFinishTimeout as e:
             self.log.info(str(e))
-            self.log.info('Canceling the compose %s', e.compose_id)
-            self.odcs_client.cancel_compose(e.compose_id)
+            for compose_id in self.all_compose_ids:
+                self.log.info('Canceling the compose %s', compose_id)
+                self.odcs_client.cancel_compose(compose_id)
```

Comment:
I'm thinking about adding it into try/except block, to not owerwrite current exception when something happen and to try cancel all composes

Reply:
Maybe "[Python raise from]" on SO is relevant?

[Python raise from]: https://stackoverflow.com/a/24752607

Reply:
wow, bleeding edge. We don't do that in enterprise ready py2.7 :eyes:

Reply:
D'OH, I keep forgetting to make appropriate time-travel settings when I come here ;p *waves at Python 2*

Reply:
try/except block has been added directly into cancel_compose method https://github.com/containerbuildsystem/atomic-reactor/blob/master/atomic_reactor/utils/odcs.py#L218

All composes will be deleted.